Shell/UIHelper.hpp

```diff
@@ -42,8 +42,7 @@ class UIHelper {
     UnitList::FIFO _units;
     SMTLIBLogic _smtLibLogic;
     bool _hasConjecture;
-    bool _proofHasConjecture;
-    LoadedPiece() : _smtLibLogic(SMT_UNDEFINED), _hasConjecture(false), _proofHasConjecture(true) {}
+    LoadedPiece() : _smtLibLogic(SMT_UNDEFINED), _hasConjecture(false) {}
```

Comment:
Could this just be C++11 assignments to the fields?

Reply:
Done!